features/data/fixtures.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO public.fleet_management_user(
+    password, is_superuser, username, first_name, last_name, email, is_staff, is_active, date_joined)
+    VALUES ('pbkdf2_sha256$120000$B4VUk7IZMZCk$t9nZBMo6q4EghFgeqkTLhstGRTlYCRfHJtGkuNzUTK0=', FALSE, 'solomiiavyshniak@eukr.net', 'Ада', 'Гавриленко', 'solomiiavyshniak@ukr.net', FALSE, TRUE, '2019-02-18');
```

Comment:
I guess we would also need to user fixtures in other scenarios. We can think of a better way to create model users before testing whole app / runninig them locally.

I would rather not use sql statements but. Plus there is (probably) real email, so we should avoid that. Let's create model users which could be used in behave, unit and manual tests as well (shared dict or something like that).

Also no newline at the end of the file.